pushapkscript/requirements/base.py38.txt

```diff
@@ -899,12 +902,13 @@ uritemplate==4.1.1 \
     # via
     #   github3-py
     #   google-api-python-client
-urllib3==1.26.8 \
-    --hash=sha256:000ca7f471a233c2251c6c7023ee85305721bfdf18621ebff4fd17a8653427ed \
-    --hash=sha256:0e7c33d9a63e7ddfcb86780aac87befc2fbddf46c58dbb487e0855f7ceec283c
+urllib3==1.26.9 \
+    --hash=sha256:44ece4d53fb1706f667c9bd1c648f5469a2ec925fcf3a776667042d645472c14 \
+    --hash=sha256:aabaf16477806a5e1dd19aa41f8c2b7950dd3c746362d7e3223dbe6de6ac448e
     # via requests
-voluptuous==0.12.2 \
-    --hash=sha256:4db1ac5079db9249820d49c891cb4660a6f8cae350491210abce741fabf56513
+voluptuous==0.13.1 \
```

Comment:
! I thought voluptuous was going to bitrot. Nothing jumping out at me https://github.com/alecthomas/voluptuous/blob/master/CHANGELOG.md